main.go

```diff
@@ -227,10 +227,9 @@ func main() {
 	}
 
 	if len(publicInstallPageMap) > 0 {
-		urlsByPage, sep := "", ""
+		urlsByPage, sep := ""\n""
```

Comment:
I think this \n was misplaced.
Did you mean:
`urlsByPage, sep := "", "\n"`

It will generate something like this:
![screen shot 2018-04-26 at 11 16 48](https://user-images.githubusercontent.com/37296013/39297578-21f72644-4944-11e8-9067-4236ef757e31.png)